### PR TITLE
Poetry fix ci

### PR DESCRIPTION
**Pull request by whabanks**
# Summary | Résumé
Hopefully fixing the CI dockerfiles with this!

### CODE REVIEW

File: ci/Dockerfile [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
 RUN python -m pip install --upgrade pip
 
 RUN set -ex && mkdir /app
 
 WORKDIR /app
 
-COPY requirements.txt /app
-RUN set -ex && pip3 install -r requirements.txt
+# COPY requirements.txt /app
+# RUN set -ex && pip3 install -r requirements.txt
```
Comment (sastels) [inline ci/Dockerfile:17]: Can delete these

File: ci/Dockerfile.lambda [hunk 2]
```diff
@@ -36,7 +37,7 @@ RUN chmod 755 /usr/bin/aws-lambda-rie /entry.sh /sync_lambda_envs.sh
 # New Relic lambda layer
 RUN unzip newrelic-layer.zip -d /opt && rm newrelic-layer.zip
 
-ENTRYPOINT [ "/entry.sh" ]
+ENTRYPOINT [ "sh", "-c", "poetry run /entry.sh" ]
```
Comment (sastels) [inline ci/Dockerfile.lambda:40]: not sure if you need the "sh", "-c" here?

File: local/Dockerfile [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
+RUN python -m pip install wheel poetry==${POETRY_VERSION}
 
 RUN set -ex && mkdir /app
 
 WORKDIR /app
 
-COPY requirements.txt /app
-RUN set -ex && pip3 install -r requirements.txt
+# COPY requirements.txt /app
+# RUN set -ex && pip3 install -r requirements.txt
```
Comment (sastels) [inline local/Dockerfile:16]: can also delete these

File: local/Dockerfile [hunk 4]
```diff
@@ -1,24 +1,27 @@
...
+RUN poetry install
+
 ENV PORT=6011
 
 ARG GIT_SHA
 ENV GIT_SHA ${GIT_SHA}
 
-CMD ["sh", "-c", "gunicorn -c gunicorn_config.py application"]
\ No newline at end of file
+CMD ["poetry", "run", "gunicorn -c gunicorn_config.py application"]
```
Comment (sastels) [inline local/Dockerfile:27]: need the "sh", "-c" here probably? Mind you, I don't think this file is used for anything.

Comment (whabanks) [reply local/Dockerfile:27]: Correct, good catch

File: ci/Dockerfile.lambda (hunk 2 above)
Comment (whabanks) [reply ci/Dockerfile.lambda:40]: I ran `docker run --entrypoint="poetry run /entry.sh" api-test-lambda` and ended up with 
```
unable to start container process: exec: "poetry run /entry.sh": stat poetry run /entry.sh: no such file or directory: unknown.
```

Comment (sastels) [reply ci/Dockerfile.lambda:40]: ah, interesting!

File: local/Dockerfile (hunk 3 above)
Comment (whabanks) [reply local/Dockerfile]: deleted!

**Review by sastels — APPROVED** [review]
Looks good!

**Pull request merged**